infra/managed-identity/managed-identity.parameters.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "managedIdentity": {
+            "value": {
+                "name": "#{{ infraResourceNamePrefix }}#{{ nc_resource_managedidentity }}#{{ nc_instance_regionid }}01"
```

Comment:
I believe the name should have a true identifying feature for it ... for examples, FFC's are like: **{namingConvention}-ahr-application-role**